vllm/compilation/fx_utils.py

```diff
@@ -42,3 +46,9 @@ def find_getitem(node: fx.Node, idx: int) -> fx.Node:
     ret = find_getitem_maybe(node, idx)
     assert ret is not None, f"Could not find getitem {idx} in node {node}"
     return ret
+
+
+# Asserts that the node only has one user and returns it
+def get_only_user(node: fx.Node) -> fx.Node:
```

Comment:
I haven't thought about this very hard, but it is possible that a node "A" has only one user but there is another node whose Tensor shares storage with node "A" that needs to be taken into account

Reply:
I can add that as a comment. But I think here we're just tracking the data flow, not storage (to match the pattern) so I think this should be fine

Reply:
added comment